schemas/produktside/innholdsseksjon/InnholdsseksjonPreview.tsx

```diff
@@ -27,7 +27,7 @@ export function InnholdsseksjonPreview(props: Props) {
         <InnholdsseksjonIkon />
         <h5>{title}</h5>
       </div>
-      <PortableText value={innhold} />
+      <BlockContentPreview blocks={innhold} color={"lightblue"} label={"Innholdsseksjon"} />
```

Comment:
BlockContentPrerview bruker den gamle @sanity/block-content-to-react
Finnes det alternativ komponent som tar i bruk det nye?

https://github.com/portabletext/react-portabletext/blob/main/MIGRATING.md

Reply:
bytter tilbake til PortableText for dette tilfellet, men vi har en jobb med å kvitte oss med den gamle modulen @sanity/block-content-to-react

Reply:
Vi kan bruke portabletext, men da vil vi miste all previewgreien for custom ting, så vi må berike portabletext med vår egen preview, derfor jeg brukte blockcontentpreview midlertidig til vi mekket vår egen. @ssaegrov